Jenkinsfile.eclipse.documentation

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+/*
+ * This is an additional Jenkinsfile, for building the documentation
+ * with the GitHub site-maven-plugin within the Eclipse Foundation infrastructure
+ */
+
+pipeline {
+    agent any
+
+    tools {
+        maven 'apache-maven-latest'
+        jdk 'adoptopenjdk-hotspot-jdk8-latest'
+    }
+
+    environment {
+        GITHUB_CREDENTIALS = credentials('7d588c11-a40f-4529-bcaa-57b6d9691af6')
+    }
+
+    stages {
+        stage('Checkout') {
+            steps {
+                git branch: 'neubs-bsi/doc/hostWithGhPages', credentialsId: '7d588c11-a40f-4529-bcaa-57b6d9691af6', url: 'https://github.com/bsinno/antenna.git'
+            }
+        }
+        stage('Configure Git') {
+            steps {
+                sh 'git config --global user.name "${GITHUB_CREDENTIALS_USR}"'
+                sh 'git config --global user.author "${GITHUB_CREDENTIALS_USR}"'
```

Comment:
To the best of my knowledge, there is no `user.author` configuration key in Git, only [author.name](https://git-scm.com/docs/git-config#Documentation/git-config.txt-authorname), but that's not needed anyway because it's what `user.name` defaults to.